components/infrastructure/modules/ROOT/pages/archive-node.adoc

```diff
@@ -75,30 +85,29 @@ skale --help
 . Prepare `.env` file:
 +
 ```
-CONTAINER_CONFIGS_STREAM="2.2.0-sync-node.0"
-ENDPOINT=[[By Validator], YOUR GETH/INFURA ENDPOINT]
-MANAGER_CONTRACTS_ABI_URL="https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/mainnet/skale-manager/1.9.0/skale-manager-1.9.0-mainnet-abi.json"
-IMA_CONTRACTS_ABI_URL="https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/mainnet/IMA/1.3.0/mainnet/abi.json"
-DISK_MOUNTPOINT=[[By Validator], your attached storage /dev/sda or /dev/xvdd (this is an example. You just need to use your 2TB block device)]
+CONTAINER_CONFIGS_STREAM="2.2.1-sync"
+ENDPOINT=[YOUR GETH/INFURA ENDPOINT]
+IMA_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/mainnet/IMA/1.5.0/mainnet/abi.json
+MANAGER_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/mainnet/skale-manager/1.9.3/skale-manager-1.9.3-mainnet-abi.json
+DISK_MOUNTPOINT=[Your attached storage /dev/sda or /dev/xvdd (this is an example. You just need to use your 2TB block device)]
 DOCKER_LVMPY_STREAM="1.0.2-stable.0"
 ENV_TYPE="mainnet"
-
-SCHAIN_NAME=[[By Validator], SCHAIN NAME FOR YOUR SYNC NODE]
+SCHAIN_NAME=[SCHAIN NAME FOR YOUR ARCHIVE NODE]
 ```
 
-. Init sync node:
```

Comment:
are we going to keep only archive + historic state option?

no docs for a regular sync node?